### PR TITLE
Add IPI dual stack v6v4 profile

### PR DESCRIPTION
**Pull request by jadhaj**
Jira https://issues.redhat.com/browse/OCPQE-17125

cc @YuLi517 @pamoedom

### CODE REVIEW

**Comment by jadhaj** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14__amd64-nightly-
baremetalds-ipi-ovn-ipv4-dualstack-v6-primary

**Comment by jadhaj** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14__amd64-nightly-baremetalds-ipi-ovn-ipv4-dualstack-v6-primary

**Comment by jadhaj** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-baremetalds-ipi-ovn-ipv4-dualstack-v6-primary

**Comment by YuLi517** [comment]
@liangxia Could you help to approve it? We will continue to build HyperShift workflow.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 1]
```diff
@@ -2328,6 +2328,20 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4
+- as: baremetalds-ipi-ovn-ipv4-dualstack-v6-primary
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:2331]: Missing `fN` in the test name to indicate its test frequency.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 2]
```diff
@@ -2328,6 +2328,20 @@ tests:
...
+  cron: 37 13 2,6,10,14,19,24,28 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v6v4
+        MIRROR_IMAGES=true
+        NETWORK_TYPE=OVNKubernetes
+      E2E_RUN_TAGS: '@amd64 and @baremetal-ipi and @network-ovnkubernetes'
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:2340]: Need `and not @fips` as this profile does not enable fips.

**Pull request merged**